File: src/utils/category_labels.py
CATEGORY_LABELS: dict[str, str] = {
    "adult": "Актрисы (18+)",
    "anime": "Аниме",
    "cartoons": "Мультфильмы",
    "movies_series": "Кино / Сериалы",
}


def category_label(category: str) -> str:
    return CATEGORY_LABELS.get(category, category)


def format_categories(categories: list[str]) -> str:
    return ", ".join(category_label(category) for category in categories)

File: webapp/miniapp-frontend/src/utils/categoryLabels.ts
const CATEGORY_LABELS: Record<string, string> = {
  adult: "Актрисы (18+)",
  anime: "Аниме",
  cartoons: "Мультфильмы",
  musicians: "Музыканты",
  movies_series: "Кино / Сериалы"
};

export function categoryLabel(category: string): string {
  return CATEGORY_LABELS[category] ?? category;
}
